server/pbenchinacan/deploy-dependencies

```diff
@@ -91,10 +93,12 @@ podman run \
   --name "${PB_POD_NAME}-keycloak" \
   --pull ${PB_KEYCLOAK_IMAGE_PULL_POLICY} \
   --restart no \
+  -v ${PB_DEPLOY_FILES}/pbench-server.crt:/opt/keycloak/conf/keycloak.crt:Z \
+  -v ${PB_DEPLOY_FILES}/pbench-server.key:/opt/keycloak/conf/keycloak.key:Z \
   -e KEYCLOAK_ADMIN=admin \
   -e KEYCLOAK_ADMIN_PASSWORD=admin \
   -d \
   ${PB_KEYCLOAK_IMAGE} \
-  start-dev --health-enabled=true --http-port=8090
+  start-dev --health-enabled=true --https-port=8090 --https-certificate-file=/opt/keycloak/conf/keycloak.crt --https-certificate-key-file=/opt/keycloak/conf/keycloak.key
```

Comment:
Maybe it's just me, but it strikes me as odd to map each of the TLS files into a particular location in the container and then to have to tell Keycloak where to find them.  Are there a default values for `--https-certificate-file` and `--https-certificate-key-file`, such that we could just map the files into the spots where Keycloak will already look for them?  

Failing that, is `/opt/keycloak/conf` a well-known location?  If not, perhaps we could do a single mount of `${PB_DEPLOY_FILES}` in a simpler unconventional location (like `/pbench_tls` or something), and then point Keycloak there.

Otherwise, I think `/etc/pki/tls/certs/` and `/etc/pki/tls/private/` are more conventional places for certificates and keys, respectively, so I would recommend locating them there instead of under `/opt`.

Reply:
Yes, `/opt/keycloak` is a very well-known location, that's where all the Keycloak related configuration lives. Take a look at this [dockerfile](https://www.keycloak.org/server/containers), the keycstore is located under `conf` sub-directory of the `/opt/keycloak`. I didnt experiment a lot with java Keystore so not sure if storing separate key and cert under the `/conf` would work. I followed this [doc](https://www.keycloak.org/server/enabletls) on enabling the SSL on Keycloak.